admin-dev/themes/new-theme/js/pages/product/edit/index.js

```diff
@@ -78,4 +79,5 @@ $(() => {
   new ProductSuppliersManager();
   new FeatureValuesManager(window.prestashop.instance.eventEmitter);
   new CustomizationsManager();
+  new CombinationsManager();
```

Comment:
Can be improved later, but it might be nice to have a way to know beforehand if the product has combinations or not And not init this component for simple products

Reply:
I'll add todo for that